Mathlib/MeasureTheory/Integral/Lebesgue/Basic.lean

```diff
@@ -272,6 +272,55 @@ theorem setLIntegral_congr_fun {f g : α → ℝ≥0∞} {s : Set α} (hs : Meas
   rw [EventuallyEq]
   rwa [ae_restrict_iff' hs]
 
+section
+
+/-- The Lebesgue integral is zero iff the function is a.e. zero. The proof implicitly uses
+Markov's inequality, but it has been inlined for the sake of imports. -/
+@[simp]
+theorem lintegral_eq_zero_iff' {f : α → ℝ≥0∞} (hf : AEMeasurable f μ) :
+    ∫⁻ a, f a ∂μ = 0 ↔ f =ᵐ[μ] 0 := by
```

Comment:
```suggestion
/-- The Lebesgue integral is zero iff the function is a.e. zero. -/
@[simp]
theorem lintegral_eq_zero_iff' {f : α → ℝ≥0∞} (hf : AEMeasurable f μ) :
    ∫⁻ a, f a ∂μ = 0 ↔ f =ᵐ[μ] 0 := by
  /- The proof implicitly uses Markov's inequality, but it has been inlined for the sake of imports. -/
```
The comment about the proof should not be in the tooltip docstring for the theorem: people using the theorem don't care how it is proved.